lms/djangoapps/teams/models.py

```diff
@@ -319,25 +319,20 @@ def get_memberships(cls, username=None, course_ids=None, team_ids=None):
         return queryset
 
     @classmethod
-    def user_in_team_for_course(cls, user, course_id, topic_id=None):
+    def user_in_team_for_teamset(cls, user, course_id, topic_id):
         """
-        Checks user membership in two ways:
-        if topic_id is None, checks to see if a user is assigned to any team in the course
-        if topic_id (teamset) is provided, checks to see if a user is assigned to a specific team in the course.
+        Using the provided teamset_id, checks to see if a user is assigned to any team in the teamset.
```

Comment:
The olde comments that were removed provide the context for topic_id (parameter) and teamset. The new comments don't have any context which could confuse future devs. I would suggest making it explicit in comments that topic_id is a synonym for teamset_id or something along those lines.